src/internal/operations/backup_test.go

```diff
@@ -276,14 +277,20 @@ func makeDetailsEntry(
 	return res
 }
 
+// TODO(ashmrtn): This should belong to some code that lives in the kopia
+// package that is only compiled when running tests.
```

Comment:
So long as it isn't referenced in a non-test file, the func is, by default, not compiled into the release binary.  If you want to additionally identify/enforce its purpose so that it doesn't accidentally get referenced, you could take in a testing.T param and call t.Helper().

Reply:
it really feels like we should be using build constraints/tags or something for stuff like that rather than requiring unused parameters

Reply:
I'm not entirely a fan of the unused parameter either, it just makes the purpose more obvious _if_ it's deemed to confusing/available.  Really though, it should be easy to catch ourselves borrowing from test helpers in live code.  For that matter, it shouldn't take much effort of any kind (besides common sense review) to keep the compiled binary clean.